cw_decred/lib/wallet.dart

```diff
@@ -619,4 +644,15 @@ abstract class DecredWalletBase
 
   @override
   String get password => _password;
+
+  @override
+  set isEnabledAutoGenerateSubaddress(bool value) {
+    this._isEnabledAutoGenerateSubaddress = value;
+    this.walletAddresses.isEnabledAutoGenerateSubaddress = value;
+  }
+
+  @override
+  bool get isEnabledAutoGenerateSubaddress {
+    return this._isEnabledAutoGenerateSubaddress;
```

Comment:
If you return `this.walletAddresses.isEnabledAutoGenerateSubaddress` here, then we won't need `this._isEnabledAutoGenerateSubaddress`.

Reply:
ahh, thanks!

Reply:
In cw_bitcoin they are doing this with two @observer bools but it doesn't seem to work